test-data/musicxml-testsuite/76a-TrackVolume.musicxml

```diff
@@ -0,0 +1,518 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move the new "custom" testfiles somewhere outside the "musicxml-testsuite". This folder is a 3rd-party MusicXML testsuite. It is not really maintained by alphaTabThis and the whole folder might be overwritten on updates.

Reply:
Ah, didn't know that. 
I shall move it later today, do you have any preference for the folder name?